examples/chef/common/chef-operational-state-delegate-impl.cpp

```diff
@@ -201,6 +205,89 @@ void OperationalState::Shutdown()
     }
 }
 
+chip::Protocols::InteractionModel::Status chefOperationalStateWriteCallback(chip::EndpointId endpointId, chip::ClusterId clusterId,
+                                                                            const EmberAfAttributeMetadata * attributeMetadata,
+                                                                            uint8_t * buffer)
+{
+    chip::Protocols::InteractionModel::Status ret = chip::Protocols::InteractionModel::Status::Success;
+    VerifyOrDie(endpointId == 1); // this cluster is only enabled for endpoint 1.
```

Comment:
Again I think you should be returning an error here and not dying. I think you can use `VerifyOrReturnValue`

Reply:
Hey so these seem to standard across all chef sample apps. Will raise this with chef-tool owners and possibly put a fix in a follow up PR.